idunn/api/places.py

```diff
@@ -49,6 +52,8 @@ def get_place(id, es: Elasticsearch, indices: IndexNames, settings: Settings, la
     loader = places.get(es_place[0].get('_type'))
 
     if loader is None:
+        prometheus.exception("FoundPlaceWithWrongType {}".format(es_place[0].get('_type')))
```

Comment:
I don't think we want the `type` in the prometheus exception, we want an aggregated metric